pillar/components/firewall.sls

```diff
@@ -35,3 +35,7 @@ firewall:
       uds:
         - 5000/tcp
         - 5000/udp
+      consul:
```

Comment:
Consul is supposed to run on private data interface. Comment these entries for now.